cpp/src/arrow/python/arrow_to_pandas.cc

```diff
@@ -371,7 +371,7 @@ class PandasWriter {
       return Status::OK();
     }
     PyAcquireGIL lock;
-
+    internal::InitPandasStaticData();
```

Comment:
Shouldn't this simply go into the `PandasWriter` constructor?

Reply:
You can simply take it if in doubt.

Reply:
wasn't sure if that was expensive.  done.